docs/builder/token-promotion.md

```diff
@@ -0,0 +1,72 @@
+# Free Microsoft Developer Account Promotion
+
+For a limited time, PWABuilder is offering free Microsoft Store Developer accounts (normally $19) to individual developers who have a progressive web app that meets specific requirements. With a Microsoft Sore Developer account, you can deploy your progressive web app to the Microsoft Store, extending your app's reach to millions of users on Windows.
+
+## Requirements 
+
+To qualify for a free Microsoft Store developer account, you must:
+
+  * own a PWA that is installable, contains all required manifest fields, and implements at least two desktop enhancements
+  * live in a country or region where the Windows program in Partner Center is offered. See here for the full list of countries
+  * have a valid Microsoft Account to use to sign up for the Microsoft Store on Windows developer account
+  * not have an existing Microsoft Store on Windows individual developer/publisher account
+  * use the Store Token to create a Microsoft Store on Windows developer account within 30 calendar days of Microsoft sending you the token, using the * * * same Microsoft Account you used to sign in here
+  * plan to publish an app in the store this calendar year (prior to 12/31/2023 midnight Pacific Standard Time)
+
+To check if your progressive web qualifies and to claim a token, follow the directions below.
+
+## Claiming a Token
+
+Before you can get started claiming a token, make sure you have a valid, public URL for your progressive web app.
+
+1. Head to [PWABuilder.com](https://www.pwabuilder.com/).
+
+2. Enter the URL to your progressive web app and hit `Start`.
+
+3. On the report card page, there should be a notification under `Action Items`:
+
+
+4. Click `Check Now`.
+
+5. Your PWA will be evaluated to see if it qualifies, if passes all tests, you can continue. Otherwise, update your PWA to match the requirements or try a different URL.
```

Comment:
Should we remove the "try a different URL". IMO, it feels possibly confusing as the dev should already know what the URL to their app is. Makes it sound like you should get a token by just putting in a random PWA you find that can meet the goals